keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -262,62 +289,94 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
             .isEqualTo(deliveryConfig.environments)
         }
 
-        test("constraint states can be retrieved and updated") {
-          val environment = deliveryConfig.environments.first { it.name == "staging" }
-          val recentConstraintState = repository.constraintStateFor(deliveryConfig.name, environment.name)
-
-          expectThat(recentConstraintState)
-            .hasSize(1)
-
-          expectThat(recentConstraintState.first().status)
-            .isEqualTo(ConstraintStatus.PENDING)
-
-          val constraint = recentConstraintState
-            .first()
-            .copy(status = ConstraintStatus.PASS)
-          repository.storeConstraintState(constraint)
-          val appConstraintState = repository.constraintStateFor(deliveryConfig.application)
-          val updatedConstraintState = repository.constraintStateFor(deliveryConfig.name, environment.name)
-
-          expectThat(appConstraintState)
-            .contains(updatedConstraintState)
-          expectThat(updatedConstraintState)
-            .hasSize(1)
-          expectThat(updatedConstraintState.first().status)
-            .isEqualTo(ConstraintStatus.PASS)
-        }
+        context("artifact constraint flows") {
```

Comment:
Indentation changes will throw off the diff viewer here. I suggest ignoring whitespace when reviewing.